04-onion-routing.md

```diff
@@ -255,9 +255,12 @@ The writer:
   - MUST include `amt_to_forward` and `outgoing_cltv_value` for every node.
   - MUST include `short_channel_id` for every non-final node.
   - MUST NOT include `short_channel_id` for the final node.
+  - MAY include application-specific [custom tlv records](01-messaging.md#type-length-value-format).
```

Comment:
I don't think that the changes in Bolt 4 are necessary.
The changes you made in Bolt 1 already imply that.
I would remove all changes to this file, wdyt?

Reply:
Bolt 1 defines the range, but I think that we should still detail the exact behavior everywhere we use tlv.

Reply:
Meh, I think this applies to everywhere we use TLV so I think it would be weird to add this line to every message requirement (which is what we'll end up doing since at some point every message will contain a tlv stream). But it's nit-picking :)

Reply:
I've come round to removing all changes in this file as you suggested. I agree it is clear enough with only the BOLT1 changes.